spec/features/accordion_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+describe "Accordion", js: true, type: :view do
+  scenario "There is a page with the accordion rendered" do
+    given_i_visit_a_page_with_the_accordion_component
+    then_the_accordion_loads
+    then_the_accordion__javascript_loads
+    when_i_click_the_first_accordion_section
+    then_the_accordion_opens
+    when_i_click_the_first_accordion_section
+    then_the_accordion_has_data_attributes
+  end
+
+  def given_i_visit_a_page_with_the_accordion_component
+    visit "/accordionexample"
+  end
+
+  def then_the_accordion_loads
+    expect(page).to have_css(".gem-c-accordion", visible: true)
```

Comment:
Using `visible: :visible` is preferred [^1], as using `false` can lead to confusion since it finds both visible _and_ invisible elements - so `false` is the equivalent to `visible: :all`, rather than being the equivalent to `visible: :hidden`.

 ```suggestion
     expect(page).to have_css(".gem-c-accordion", visible: :visible)
```


[^1]: https://docs.rubocop.org/rubocop-rspec/cops_rspec/capybara.html#rspeccapybaravisibilitymatcher

Reply:
huh, thank you - really good to know.